src/test/owr/pectra/PectraWithdrawalMock.sol

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.19;
+
+import "forge-std/console.sol";
+
+contract PectraWithdrawalMock {
```

Comment:
Similar to other places, maybe this should be called an `ExecutionLayerWithdrawalSystemContractMock`